tests/linux_kernel/helpers/test_slab.py

```diff
@@ -75,6 +76,31 @@ def test_slab_cache_is_merged_true(self):
             self.fail("couldn't find slab cache")
         self.assertTrue(slab_cache_is_merged(slab_cache))
 
+    def test_slab_cache_aliases(self):
+        alias_to_name = get_slab_cache_aliases(self.prog)
+        for aliases in self._slab_cache_aliases().values():
+            # Some entries in sysfs are symlinks, and some are not.
+            # Ignore alias groups which are of length 1 -- either they
+            # refer to symlinks but there's only one cache mapping to them, in
+            # which case it doesn't matter if they appear in the mapping, or
+            # they are not symlinks, and won't show up in the mapping anyway.
+            if len(aliases) <= 1:
```

Comment:
Now that you removed entries with that only alias to themselves, maybe we should test that explicitly? E.g.,

```python3
if len(aliases) == 1:
    self.assertNotIn(aliases[0], alias_to_name)
else:
    ... your original test ...
```

Reply:
I also forgot to assert that, for len(aliases) >= 1, the target alias should not be present as a key in the dict. Added that in my latest revision too.